frontend/packages/ceph-storage-plugin/src/components/modals/volume-snapshot-modal/_volume-snapshot-modal.scss

```
@@ -4,7 +4,7 @@
     margin-right: var(--pf-global--spacer--md) !important;
   }
 
-.ceph-volume-snapshot-modal__input--count {
+.ceph-volume-snapshot-modal__input--dropdown {
```

`dropdown` is not a valid modifier. 
```suggestion
.ceph-volume-snapshot-modal-input__dropdown {
```